Survey/Configurations/Environment/Environment.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Environment.swift
+//  Survey
+//
+//  Created by Khanh on 01/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+enum Environment: String {
+
+    case development = "Debug Production"
+    case staging = "Debug Staging"
```

Comment:
Should this be no space?
```suggestion
    case development = "DebugProduction"
    case staging = "DebugStaging"
```

Reply:
I used your suggestion, so I removed this definition at https://github.com/nkhanh44/ic-survey-ios/pull/14/commits/2809248129b9cadb424df423d57d6fdf3722b046